applications/asset_tracker/src/main.c

```diff
@@ -229,14 +229,16 @@ void error_handler(enum error_type err_type, int err_code)
 #if defined(CONFIG_LTE_LINK_CONTROL)
 		/* Turn off and shutdown modem */
 		int err = lte_lc_power_off();
-		__ASSERT(err == 0, "lte_lc_power_off failed: %d", err);
-#endif
+		if (err) {
+			printk("Could not shut down LTE link gracefully\n");
```

Comment:
Could not shut down **_the_** (..)

Is "gracefully" a common term that is understood across users?

Reply:
Perhaps not. I'll update it anyway so that also people who are not that into grace will understand :)